rand.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
```

Comment:
Is this copyright notice correct? We might need to preserve the original. (Maybe both?)

Reply:
Hmm, this is a new file, so 2018 is correct, right? Or did you mean something else?

Reply:
I meant the copyright holder. There's code in here copied from the upstream Go codebase right?

Reply:
Ah, hmm. Sure, what should I change it to?

Reply:
I don't know what the Word of God is on these things. Maybe just copy-paste both notices at the top?

Reply:
Found [this](https://softwareengineering.stackexchange.com/questions/264361/how-to-license-open-source-code-you-have-heavily-modified-under-the-bsd-license) and followed its advice.